index.js

```diff
@@ -118,20 +118,21 @@ class Autohook extends EventEmitter {
     token_secret = (process.env.TWITTER_ACCESS_TOKEN_SECRET || '').trim(),
     consumer_key = (process.env.TWITTER_CONSUMER_KEY || '').trim(),
     consumer_secret = (process.env.TWITTER_CONSUMER_SECRET || '').trim(),
+    ngrok_secret = (process.env.NGROK_AUTH_TOKEN || '').trim(),
     env = (process.env.TWITTER_WEBHOOK_ENV || '').trim(),
     port = process.env.PORT || DEFAULT_PORT,
     headers = [],
   } = {}) {
 
-    Object.entries({token, token_secret, consumer_key, consumer_secret, env, port}).map(el => {
+    Object.entries({token, token_secret, consumer_key, consumer_secret,ngrok_secret, env, port}).map(el => {
       const [key, value] = el;
       if (!value) {
         throw new TypeError(`'${key}' is empty or not set. Check your configuration and try again.`);
       }
     });
 
     super();
-    this.auth = {token, token_secret, consumer_key, consumer_secret};
+    this.auth = {token, token_secret, consumer_key, consumer_secret ,ngrok_secret};
```

Comment:
Style: remove space between variable name and `,`; add a space between `,` and the variable name:

```js
this.auth = {token, token_secret, consumer_key, consumer_secret, ngrok_secret};
```

Note that `this.auth` carries Twitter OAuth tokens, and the object is structured so that it can be consistently used across other areas of Autohook that detail with Twitter's OAuth authentication. Ideally, `ngrok_secret` should be an instance variable (i.e. `this.ngrokSecret`, same level as `this.env` or `this.port`); this way we can keep the ngrok tokens logically separated from OAuth tokens.